resources/components/ImageCard.vue

```diff
@@ -19,17 +19,37 @@
 				</ul>
 			</div>
 
-			<div class="wbmad-buttons">
-				BUTTONS GO HERE
+			<div class="wbmad-action-buttons">
+				<base-button
+					v-bind:classes="[ 'wbmad-action-buttons__publish' ]"
```

Comment:
This syntax sucks and I'd prefer to make this a string with a space between each class, just like HTML, but I couldn't figure out a way to make that work in the button component